app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -103,7 +110,7 @@
  */
 @SuppressWarnings({"PMD.ExcessiveClassLength", "PMD.CyclomaticComplexity", "PMD.TooManyMethods",
         "PMD.ModifiedCyclomaticComplexity", "PMD.TooManyFields", "PMD.StdCyclomaticComplexity" })
-public class BrowserFragment extends WebFragment implements View.OnClickListener, DownloadDialogFragment.DownloadDialogListener, View.OnLongClickListener {
+public class BrowserFragment extends WebFragment implements LifecycleObserver, View.OnClickListener, DownloadDialogFragment.DownloadDialogListener, View.OnLongClickListener, BiometricAuthenticationDialogFragment.BiometricAuthenticationListener {
```

Comment:
\[PMD\] INFO: The class 'BrowserFragment' has a NCSS line count of 702 \(Highest = 49\)\.